snappy/security.go

```diff
@@ -71,6 +71,12 @@ var (
 		SecurityCaps: []string{},
 	}
 
+	// the default migration skill if no default is uses in
```

Comment:
That doesn't sound right. We don't want the migration-skill to come in by default. We want people to explicitly mention it if they depend on anything non-default.

The permissions we discussed so far are:

- Writing into the applications own space. That should definitely come by default. It did so in the old convention, and it should continue to do so in the new convention.

- Network client access. That used to be a default in the old convention when nothing was specified, and we agreed to not have it by default in the new convention, at least for now. If people want network access, they need to explicitly mention migration-skill. That ensures we can go into either direction soon depending on what we decide at the sprint, without having to break anyone.

Is there anything else we're missing that exists in the default template of the old convention?